wakusim.env

```diff
@@ -1,5 +1,5 @@
 # Env variables for metal-01.he-eu-hel1.wakusim.misc host.
-NWAKU_IMAGE=wakuorg/nwaku:latest
+NWAKU_IMAGE=quay.io/wakuorg/nwaku-pr:2607-rln-v2
```

Comment:
ideally we should point to latest master (with rlnv2). do we have such image?

Reply:
I will find out where the latest master image with RLNv2 can be found